snews_cs/simulate.py

```diff
@@ -2,11 +2,11 @@
 module to simulate observation messages
 and trigger alerts
 """
-
+# Do we need this ??
```

Comment:
I think we don't.
It still works, but I think we have a better tool to trigger alerts instead of this random generator.
Also, it probably needs adjusting, so we can go ahead and dump it